third_party/2/simplejson/__init__.pyi

```diff
@@ -1,6 +1,6 @@
 from typing import Any, IO
 
-def dumps(obj: Any) -> str: ...
+def dumps(obj: Any, fp: IO[str], *args: Any, **kwds: Any) -> str: ...
```

Comment:
This is incorrect. Looking at the source code of simplejson 3.10.0, this is how `dumps` looks like:
```
def dumps(obj, skipkeys=False, ensure_ascii=True, check_circular=True,
          allow_nan=True, cls=None, indent=None, separators=None,
          encoding='utf-8', default=None, use_decimal=True,
          namedtuple_as_object=True, tuple_as_array=True,
          bigint_as_string=False, sort_keys=False, item_sort_key=None,
          for_json=False, ignore_nan=False, int_as_string_bitcount=None,
          iterable_as_array=False, **kw):
```

Note: there's actually quite a few explicit arguments you could list. But there is **no** mandatory `fp` argument.